config.json

```diff
@@ -201,6 +201,23 @@
                 "MAX_LIMIT": 2,
                 "WINDOW_SIZE": 1440
             }
+        },
+        {
+            "ID": "ECLIPSET",
+            "NAME": "Eclipse Testnet",
+            "TOKEN": "ECLP",
+            "RPC": "https://subnets.avax.network/eclipsecha/testnet/rpc",
+            "CHAINID": 555666,
+            "EXPLORER": "https://subnets-test.avax.network/eclipsecha",
+            "IMAGE": "https://raw.githubusercontent.com/Thor-Financial/Eclipse-assets/main/eclipse-logo.png",
+            "MAX_PRIORITY_FEE": "300000000000",
```

Comment:
Priority fee (300 gwei) shouldn't be more than max fee (100 gwei).

Reply:
10 gwei should be sufficient here (that's what we use on C-Chain).

Reply:
Ah okay, sure will do.